src/main/java/tds/testpackageconverter/converter/mappers/TestPackageBlueprintMapper.java

```diff
@@ -289,28 +321,38 @@ private static BlueprintElement mapAssessmentBlueprintElements(final Testspecifi
                 })
                 .collect(Collectors.toCollection(LinkedHashSet::new));
 
-        claimBlueprintElements.forEach(blueprintElement -> mapBlueprintElement(blueprintElement, parentIdToBpElement, scoringMap));
+        claimBlueprintElements.forEach(blueprintElement ->
+                mapBlueprintElement(blueprintElement, parentIdToBpElement, scoringMap, csidEnhancer));
 
         return new ArrayList<>(claimBlueprintElements);
```

Comment:
Feel free to ignore this comment but if this method was changed to return a Collection rather than a List this `new ArrayList` would not be necessary.